docs/core/DelegationManager.md

```diff
@@ -43,7 +43,7 @@ Registers the caller as an Operator in EigenLayer. The new Operator provides the
 * Sets `OperatorDetails` for the Operator in question
 * Delegates the Operator to itself
 * If the Operator has deposited into the `EigenPodManager` and is not in undelegation limbo, the `DelegationManager` adds these shares to the Operator's shares for the beacon chain ETH strategy.
-* For each of the three strategies in the `StrategyManager`, if the Operator holds shares in that strategy they are added to the Operator's shares under the corresponding strategy.
+* For each of the three strategies in the `StrategyManager`, if the staker is not in undelegation limbo and holds shares in that strategy, then they are added to the Operator's shares under the corresponding strategy.
```

Comment:
I've been capitalizing "Staker" and "Operator" because they're "vocab" i defined in `RolesAndActors.md`

Reply:
I gave this a shot in a commit https://github.com/Layr-Labs/eigenlayer-contracts/pull/208/commits/e77dd42906faa166eae58c8583448809752602a1
Feels like an uphill battle, but I'll try to be consistent with this going forwards